benchmark/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "benchmark",
-  "version": "6.0.4",
+  "version": "6.0.5",
```

Comment:
What do you think about either not bumping it if not necessary or just aligning with the release? 🤔

Reply:
Not bumping then